reader-osm/src/main/java/com/graphhopper/reader/osm/GraphHopperOSM.java

```diff
@@ -44,4 +61,32 @@ public GraphHopperOSM setOSMFile(String osmFileStr) {
         super.setDataReaderFile(osmFileStr);
         return this;
     }
+
+    @Override
+    protected void loadOrPrepareLM() {
+        if (!getLMFactoryDecorator().isEnabled() || getLMFactoryDecorator().getPreparations().isEmpty())
+            return;
+
+        if (landmarkSplittingFeatureCollection != null && !landmarkSplittingFeatureCollection.getFeatures().isEmpty()) {
```

Comment:
This should not break any existing code. As the landmarkSplittingFeatureCollection will be null by default.